### PR TITLE
Fix db RepositoryBuild model attribute name used

### PR DESCRIPTION
**Pull request by kleesc**
**Issue:** https://issues.redhat.com/browse/PROJQUAY-1011

**Changelog:** Fix attribute used to access build namespace

**Docs:** 

**Testing:** 
- Navigate to Quay UI -> Super user panel -> Build logs -> Get Logs

**Details:**

### CODE REVIEW

File: endpoints/api/superuser_models_pre_oci.py [hunk 1]
```diff
@@ -75,7 +75,7 @@ def get_repository_build(self, uuid):
         except model.InvalidRepositoryBuildException as e:
             raise InvalidRepositoryBuildException(str(e))
 
-        repo_namespace = build.repository_namespace_user_username
+        repo_namespace = build.repository.namespace_user.username
         repo_name = build.repository_name
```
Comment (kurtismullins) [inline endpoints/api/superuser_models_pre_oci.py:79]: For the sake of consistency, would it make sense to also change the next line?
```
repo_name = build.repository.name
```

Comment (kleesc) [reply endpoints/api/superuser_models_pre_oci.py:79]: A repo name and a repo namespace are 2 different things. The latter refers to the user/org name

Comment (kleesc) [reply endpoints/api/superuser_models_pre_oci.py]: Updated to to user the correct attribute name.

**Review by kurtismullins — APPROVED** [review]
LGTM

**Pull request merged**